editor/src/messages/input_mapper/input_mappings.rs

```diff
@@ -401,7 +401,8 @@ pub fn input_mappings() -> Mapping {
 		entry!(KeyDown(Tab); modifiers=[Control], action_dispatch=PortfolioMessage::NextDocument),
 		entry!(KeyDown(Tab); modifiers=[Control, Shift], action_dispatch=PortfolioMessage::PrevDocument),
 		entry!(KeyDown(KeyW); modifiers=[Accel], action_dispatch=PortfolioMessage::CloseActiveDocumentWithConfirmation),
-		entry!(KeyDown(KeyO); modifiers=[Accel], action_dispatch=PortfolioMessage::OpenDocument),
+		entry!(KeyDown(KeyW); modifiers=[Accel,Alt], action_dispatch=PortfolioMessage::CloseAllDocumentsWithConfirmation),
+		entry!(KeyDown(KeyO); modifiers=[Accel,Shift], action_dispatch=PortfolioMessage::OpenDocument),
```

Comment:
I'm not sure why you have added an additional shortcut for opening a document. This was already bound to <kbd>Ctrl</kbd> + <kbd>O</kbd>.